plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1034,7 +1158,10 @@ void producer_plugin::set_program_options(
    producer_options.add_options()
       ("enable-stale-production,e", boost::program_options::bool_switch()->notifier([this](bool e){my->_production_enabled = e;}),
        "Enable block production, even if the chain is stale.")
-         ("pause-on-startup,x", boost::program_options::bool_switch()->notifier([this](bool p){my->_pause_production = p;}), "Start this node in a state where production is paused")
+         ("pause-on-startup,x", boost::program_options::bool_switch()->notifier([this](bool p){my->_pause_production = p;}),
+          "Start this node in a state where production is paused")
+         ("production-pause-vote-timeout-ms", bpo::value<uint32_t>()->default_value(config::default_production_pause_vote_timeout_ms),
```

Comment:
I am debating what's a best name for this. It sounds like `pause vote` not `pause production`. Maybe just `vote-timeout-ms` as the name, and the description describe what option does in details?

Reply:
This name was decided in our meeting. We wanted `production-pause-*` as we might add new `production-pause-*` options in the future.